src/components/FilterSizes.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import size from '../img/size.png';
+import './style/FilterSizes.css';
+
+
+
+class FilterSizes extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            text: false,
+            clicked: [false, false, false, false, false],
+        };
+    }
+
+    changeText = (text, key) => {
+        let arr = [false, false, false, false, false];
+        arr[key] = true;
+        this.setState({text: text, clicked: arr});
+    }
+    
+    render () {
+        let params = [ //text, style.top, style.left
+            [<><p>Notice how our input, a 6 x 6 RGB image, is split into three 6 x 6 layers.</p></>, "90%", "12%"],
```

Comment:
Is there a reason that some of these are wrapped in react fragments (e.g. if you remove the `<> </>` I think it should still work?)

Reply:
The elements with multiple `<p>` tags need to be wrapped in fragments, but I just now removed them for the single `<p>` tags.